runtime/runtime/src/lib.rs

```diff
@@ -1449,6 +1447,233 @@ impl Runtime {
             &mut metrics,
         )?;
 
+        // Step 4: process receipts.
+        let process_receipts_result = self.process_receipts(
+            protocol_version,
+            apply_state,
+            epoch_info_provider,
+            &mut outcomes,
+            &mut receipt_sink,
+            &mut state_update,
+            &mut stats,
+            &mut total,
+            &mut metrics,
+            &mut prefetcher,
+            &local_receipts,
+            &mut delayed_receipts,
+            incoming_receipts,
```

Comment:
mini nit: I would suggest ordering the arguments in such a way that: 
* inputs are first, outputs (or muts) are second
* the more general arguments go first and the more specific go later (e.g. protocol_version before receipts)

Totally optional :)